ui/src/app/pages/analysis/gke-app-launcher.spec.tsx

```diff
@@ -84,18 +91,67 @@ const setupEmptyUserStore = () => {
   };
 };
 
-it('Should display iframe with rStudio App Url', async () => {
-  setup();
-  // Confirm IFrame exist
-  expect(screen.getByTitle('RStudio embed')).toBeInTheDocument();
-  // Confirm IFrame source is same as proxyUrl for Rstudio L24
-  expect(screen.getByTitle('RStudio embed').getAttribute('src')).toBe(
-    RSTUDIO_FAKE_URL
-  );
-});
+test.each([
```

Comment:
optional: can use a `describe.each()` block instead, with the two tests inside it

Reply:
there are other test as well that doesnt require test.each so i am going to leave it like this. We can discuss more on monday

Reply:
no need, fine as-is